google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
IMO there's no better way to convey that no callback is being passed than using a no-arg value.
Conventions on passing null's around can be very lax and hard to understand without context. Not to mention that explicitly passing null into a method smells like NPE. The gradual move to Optionals would be another argument against explicitly passing in null.

In your place, I'd definitely create a new method. Up to you!